inception-project/inception
# #1951 - Custom error page

**What's in the PR**
- Replace the Spring Whitelabel error page with a nice page that uses the application design and provides a bit of information that may be helpful when reporting an error.

**How to test manually**
* For a 404 error, try e.g. accessing `/assets/blah`
* Try accessing "/whoops/test" after logging in for an error with a stack trace (make sure to be running in Wicket development mode, otherwise the test page is not mounted)

**Automatic testing**

**Documentation**
